README.md

```diff
@@ -2,10 +2,8 @@
 
 ## Installation instructions
 1. Clone the MLflow Pipelines template repo locally: `git clone https://github.com/mlflow/mlp-regression-template.git`.
-2. Checkout the latest MLflow: `git clone https://github.com/mlflow/mlflow.git`
-3. Install pipeline related requirements in MLflow. `cd mlflow && pip install -e .[pipelines]`
-4. Enter the root of the template: `cd mlp-regression-template`
-5. Install template dendencies: `pip install -r requirements.txt`
+2. Enter the root of the template: `cd mlp-regression-template`
```

Comment:
Can we tell folks to install `mlflow[pipelines]` separately? I think we should familiarize them with the installation procedure for MLP